scripts/fuzz_opt.py

```diff
@@ -704,10 +700,25 @@ def write_commands(commands, filename):
     ["--vacuum"],
 ]
 
+# Exception handling does not run with these options. We split these from
+# basic_opt_choices to handle EH fuzzing.
+advanced_opt_choices = [
```

Comment:
how about `basic_opt_choices` and `flatten_using_opt_choices`?

Reply:
Or `flattening_opt_choices`

Reply:
Changed to `flattening_opt_choices`.